BaseTools/Conf/build_rule.template

```diff
@@ -711,3 +718,37 @@
         "$(RC)" $(RC_FLAGS) $(OUTPUT_DIR)(+)$(MODULE_NAME)hii.rc ${dst}
     <Command.XCODE>
         "$(GENFW)" -o $(OUTPUT_DIR)(+)$(MODULE_NAME)hii.rc -g $(MODULE_GUID) --hiibinpackage $(HII_BINARY_PACKAGES) $(GENFW_FLAGS)
+
+# MU_CHANGE [BEGIN]

```

Comment:
As noted offline, will need to reconcile with similar changes in https://github.com/microsoft/mu_basecore/pull/942.

Reply:
I plan on letting #942 merge and fixing during a rebase

Reply:
Dropped in favor of https://github.com/microsoft/mu_basecore/pull/942